packages/@lwc/synthetic-shadow/src/faux-shadow/html-element.ts

```diff
@@ -170,3 +204,29 @@ defineProperties(HTMLElement.prototype, {
         configurable: true,
     },
 });
+
+// Note: Firefox does not have outerText, https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/outerText
+if (outerTextGetter !== null && outerTextSetter !== null) {
+    defineProperty(HTMLElement.prototype, 'outerText', {
+        get(this: Element): string {
+            if (!featureFlags.ENABLE_ELEMENT_PATCH) {
+                if (isNodeShadowed(this) || isHostElement(this)) {
+                    return getInnerText.call(this);
```

Comment:
this is outerText, why are you callling getInnerText here? Also, I could not fine the outer text implementation anymore in this diff.

Reply:
There's no spec for outerText. From [mdn](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/outerText):

> HTMLElement.outerText is a non-standard property. As a getter, it returns the same value as Node.innerText. As a setter, it removes the current node and replaces it with the given text.